src/foobar/tests/test_views.py

```diff
@@ -93,26 +93,34 @@ def test_wallet_management(self, mock_deposit_withdrawal, mock_correction):
     def test_edit_profile(self, mock_update_account):
         account_obj = factories.AccountFactory.create()
         token = signing.dumps({'id': str(account_obj.id)})
-        url = reverse('edit_profile', kwargs={'token': token})
-        bad_token = reverse('edit_profile', kwargs={'token': 'bad'})
-        response1 = self.client.get(url)
-        response2 = self.client.get(bad_token)
+        good_token_url = reverse('edit_profile', kwargs={'token': token})
+        bad_token_url = reverse('edit_profile', kwargs={'token': 'bad'})
 
         # Assert that page can be found
-        self.assertEqual(response1.status_code, 200)
-        self.assertEqual(response2.status_code, 200)
+        response = self.client.get(good_token_url)
+        self.assertEqual(response.status_code, 200)
+        response = self.client.get(bad_token_url)
+        self.assertEqual(response.status_code, 400)
```

Comment:
**SLIGHT SIDE NOTE**

Sending a `400_BAD_REQUEST` here might be slightly misleading, as we're validating query params.

The definition of when to use `400` is very broad though and as follows:
```
The server cannot or will not process the request due to an apparent client error (e.g., malformed request syntax, too large size, invalid request message framing, or deceptive request routing).
```

It might be a little bit of an overkill, however a more pinpointed status could be `422 Unprocessable Entity`(subset of `400_BAD_REQUEST`) with definition:
```
The request was well-formed but was unable to be followed due to semantic errors.
```